src/states_screens/ghost_replay_selection.cpp

```diff
@@ -457,13 +461,29 @@ void GhostReplaySelection::onColumnClicked(int column_id)
     else
         assert(0);
 
-    printf("column_id is %d\n", column_id);
-
     /** \brief Toggle the sort order after column click **/
     m_sort_desc = !m_sort_desc;
+
+    ReplayPlay::get()->sortReplay(m_sort_desc);
+
     loadList();
 }   // onColumnClicked
 
+// ----------------------------------------------------------------------------
+/** Apply the default sorting to the replay list
+ */
+void GhostReplaySelection::defaultSort()
+{
+    ReplayPlay::setSortOrder(ReplayPlay::SO_TIME);
```

Comment:
sorting multiple times in a row is not the optimal way to do it, though I guess the difference would be tiny